installation.md

```diff
@@ -1,15 +1,42 @@
-The following document is a step-by-step guide to run BWS in cluster mode.
+The following document is a step-by-step guide to run BWS.
 
-Configuration for all required modules can be specified in https://github.com/bitpay/bitcore-wallet-service/blob/master/config.js
+### Prerequisites
+Ensure MongoDB (2.6+) is installed and running. This document assumes that mongod is running at the default port 27017.
+See the configuration section to configure a different host/port.
 
+### Install BWS from NPM
+Use the following steps to Install BWS from the npmjs repository and run it with defaults.
+```bash
+npm install bitcore-wallet-service
+cd bitcore-wallet-service
+```
+To change configuration before running, see the Configuration section.
+```bash
+npm start
+```
 
-###Install BWS
+### Install BWS from github source
+Use the following steps to Install BWS from github source and run it with defaults.
 ```bash
-npm install bws
-cd bws
-````
+git clone https://github.com/bitpay/bitcore-wallet-service.git
+cd bitcore-wallet-service
+npm install
+```
+To change configuration before running, see the Configuration section.
+```bash
+npm start
+```
+### Configuration
+Configuration for all required modules can be specified in https://github.com/bitpay/bitcore-wallet-service/blob/master/config.js
 
-###Start MongoDB
+BWS is composed of 5 separate node services -
+Locker - locker/locker.js
+Message Broker - messagebroker/messagebroker.js
+Blockchain Monitor - bcmonitor/bcmonitor.js (This service talks to the Insight service hosted by Bitpay)
```

Comment:
Actually, `bcmonitor` will use whatever instance is specified in `config.js` under `blockchainExplorerOpts`